src/natcap/invest/urban_mental_health.py

```diff
@@ -0,0 +1,1335 @@
+                "to ensure correct edge pixel calculation, as InVEST will "
+                "buffer your AOI by the `search_radius` to determine the "
+                "processing area. Final outputs will be clipped to this AOI."),
+            projected=True,
+            projection_units=u.meter
+        )),
+        spec.SingleBandRasterInput(
+            id="population_raster",
+            name=gettext("population"),
+            about=gettext(
+                "Gridded population data representing the number of people "
+                "per pixel, ideally at the pixel level. Where available, "
+                "additional demographic attributes (e.g., age, income) can "
+                "enhance model accuracy and equity assessments. For "
+                "population count rasters, users are recommended to use the "
+                "WorldPop data, which has global coverage at 100 meter "
+                "resolution https://hub.worldpop.org/project/categories?id=3"
+            ),
+            data_type=int,
+            units=u.people,
+            projected=True,
+            projection_units=u.meter
+        ),
+        spec.NumberInput(
+            id="search_radius",
+            name=gettext("search radius"),
+            about=gettext(
+                "Distance used to define the surrounding area of a person's "
+                "residence that best represents daily exposure to nearby "
+                "nature."),
+            units=u.meter,
+            expression="value > 0"
+        ),
+        spec.NumberInput(
+            id="effect_size",
+            name=gettext("health effect size"),
+            about=gettext(
+                "Health indicator-specific effect size representing the "
+                "relationship between nature exposure and the mental health "
+                "outcome, given as relative risk associated with a 0.1 "
+                "increase in NDVI. If the user has an effect size value "
+                "as an odds ratio, see User's Guide."
+            ),
+            units=None, #todo: check
+            expression="value > 0"
+        ),
+        spec.VectorInput(
+            id="baseline_prevalence_vector",
+            name=gettext("baseline prevalence"),
+            about=gettext(
+                "Baseline prevalence (or incidence) rates of specific mental "
+                "health outcomes (e.g., depression or anxiety) across "
+                "administrative units within the study area. This data allows "
+                "the model to estimate preventable cases by comparing current "
+                "rates with those projected under improved nature exposure "
+                "scenarios."),
+            geometry_types={"MULTIPOLYGON", "POLYGON"},
+            fields=[
+                spec.RatioInput(
+                    id="risk_rate",
+                    about=gettext("Health risk rate")
+                )
+            ],
+            projected=None
```

Comment:
I think the default is `None`, so you could remove this line if you wanted.

Reply:
Same for a few inputs below too. But, understandable if you think it's better to be explicit!